.github/workflows/e2e.yml

```diff
@@ -40,10 +60,19 @@ jobs:
           )
     strategy:
       matrix:
-        module: [mysql-plugin, postgresql-plugin]
+        module: [mysql-plugin, postgresql-plugin, oracle-plugin]
       fail-fast: false
     steps:
       # Pinned 1.0.0 version
+      - name: Run Tests
+        if: github.event_name == 'workflow_dispatch' || github.event_name == 'push' || steps.filter.outputs.e2e-test == 'true'
+        run: python3 e2e/src/main/scripts/run_e2e_test.py --module ${{ matrix.module }}
+        env:
+          ORACLE_HOST: ${{ needs.secretsmanager.outputs.ORACLE_HOST }}
+          ORACLE_USERNAME: ${{ needs.secretsmanager.outputs.ORACLE_USERNAME }}
+          ORACLE_PASSWORD: ${{ needs.secretsmanager.outputs.ORACLE_PASSWORD }}
+          ORACLE_PORT: ${{ needs.secretsmanager.outputs.ORACLE_PORT }}
```

Comment:
There is already `Run required tests` and `Run all tests` steps below. Why do we need this extra step?

We should be able to add these env variables in both of these steps.